src/Moryx.ControlSystem/Processes/IProcessControlV10.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2025, Phoenix Contact GmbH & Co. KG
+
+using Moryx.AbstractionLayer;
+
+namespace Moryx.ControlSystem.Processes
+{
+    /// <summary>
+    /// Actions to report a running process
+    /// </summary>
+    public enum ReportAction
+    {
+        /// <summary>
+        /// The process is broken or damaged
+        /// </summary>
+        Broken,
+        /// <summary>
+        /// The process was physically removed
+        /// </summary>
+        Removed
+    }
+
+    /// <summary>
+    /// Facade interface to get more information from the control system.
+    /// </summary>
+    public interface IProcessControlV10 : IProcessControl
+    {
+
+        /// <summary>
+        /// Reports a process as broken or removed
```

Comment:
Report a specific <see ... ReportAction> to have been executed on the <param .. process>